### PR TITLE
Create ELECTIONS.md

### PR DESCRIPTION
**Pull request by tdcmeehan**

### CODE REVIEW

File: elections/ELECTIONS.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# TSC Elections Process
+## Guidelines
+1. Avoid scheduling around major holidays.  Use best judgement to determine where TSC members are located and which dates they may be unavailable.
+2. Announce the election in the [prestodb/presto main repo README](https://github.com/prestodb/presto)
```
Comment (majetideepak) [inline elections/ELECTIONS.md:4]: Should we formally announce it on Slack as well?

File: elections/ELECTIONS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Process
+1. The facilitator will announce the following dates:
+    1. D: the date that nominations will be open; a PR will be opened updating the PrestoDB README announcing the election; and an issue will be opened announcing the election in the TSC repo.
+    2. D+7: the date that nominations will be closed; a comment will be made in the issue announcing current candidates and any invalid candidates.
```
Comment (majetideepak) [inline elections/ELECTIONS.md:9]: probably say `; a comment will be made in the issue announcing candidates with valid and invalid nominations`?
Should we notify invalid nominations to candidates over email as well?

File: elections/ELECTIONS.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    3. D+8: the date that the election will start
+    4. D+15: the date that the election will be closed
+    5. D:16: the date that the winners are announced and a PR is opened removing the README addition to the PrestoDB repo.
+2. Fill out the self-nomination template and send to operations@prestodb.io: [tsc/SELF_NOMINATION_TEMPLATE.md](https://github.com/prestodb/tsc/blob/master/elections/SELF_NOMINATION_TEMPLATE.md)
+    1. If there are circumstances that prevent the person being nominated from self-submitting, please explain in the email with a date that the person can confirm their nomination.
+3. Before confirming the election, the facilitator will follow up with individuals who submitted “invalid” nominations (for example, external nominations that were never confirmed by the person being nominated).  The facilitator will give 24 hours for those nominations to be corrected.
+4. Details on voting procedure to be added
+5. The winners are announced in the presto-dev@lists.prestodb.io
```
Comment (majetideepak) [inline elections/ELECTIONS.md:17]: Probably add that the winners will also be announced on Slack

File: elections/ELECTIONS.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# TSC Elections Process
+## Guidelines
+1. Avoid scheduling around major holidays.  Use best judgement to determine where TSC members are located and which dates they may be unavailable.
+2. Announce the election in the [prestodb/presto main repo README](https://github.com/prestodb/presto)
+
+## Process
+1. The facilitator will announce the following dates:
+    1. D: the date that nominations will be open; a PR will be opened updating the PrestoDB README announcing the election; and an issue will be opened announcing the election in the TSC repo.
+    2. D+7: the date that nominations will be closed; a comment will be made in the issue announcing current candidates and any invalid candidates.
+    3. D+8: the date that the election will start
```
Comment (majetideepak) [inline elections/ELECTIONS.md:10]: Should the related issue be closed now?

**Review by majetideepak — COMMENTED** [review]
@tdcmeehan some thoughts below.

File: elections/ELECTIONS.md (hunk 2 above)
Comment (tdcmeehan) [reply elections/ELECTIONS.md:9]: SG

**Comment by tdcmeehan** [comment]
@majetideepak thanks for your comments, please have a look!

**Review by majetideepak — APPROVED** [review]

**Pull request merged**